### PR TITLE
TC 13-14

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/14-gods-great-love.adoc [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 
-"`For God so loved the world, that he gave his only begotten Son,
-that whosoever believed in him might not perish, but have everlasting life.`" John 3:16.
+[quote.scripture, , John 3:16]
+____
+For God so loved the world, that he gave his only begotten Son,
+that whosoever believed in him might not perish, but have everlasting life.
+____
 
-=== PREFACE.
```
Comment (Henderjay) [inline journal/modernized/14-gods-great-love.adoc:11]: I think I prefer to have prefaces centered. I know that the default heading 3 is left justified, but when it's a title of a section, I think it almost always looks better to put the [.centered] tag over it. 👍🏻

**Review by Henderjay — CHANGES_REQUESTED** [review]
just one little thing to tweak here

File: journal/modernized/14-gods-great-love.adoc (hunk 1 above)
Comment (maijapapaya) [reply journal/modernized/14-gods-great-love.adoc:11]: No problem, I agree.

**Comment by Henderjay** [comment]
merged

**Pull request merged**